src/main/webapp/components/results/results.stories.js

```diff
@@ -7,13 +7,13 @@ import genResults from '../../gen-results'
 const stories = storiesOf('Result', module)
 
 const attributes = [
-  'Title',
-  'Thumbnail',
-  'Description',
-  'Created',
-  'Modified',
-  'Checksum',
-  'ID',
+  'title',
+  'thumbnail',
+  'description',
+  'created',
+  'modified',
+  'checksum',
+  //  'id',
```

Comment:
Why is id commented out?

Reply:
I commented that out for testing. Wasn't supposed to commit that.
Good catch. Thank you

Reply:
fixed